std/meta.d

```diff
@@ -1210,6 +1211,147 @@ template aliasSeqOf(alias range)
     }
 }
 
+/**
+ * Applies the sequence of arguments `args` to the function or template `func`
+ * either as function or template arguments, at compile-time or at run-time
+ * (if CTFE is not possible).
```

Comment:
The introduction paragraph should answer the obvious question "What's wrong with `func(args)`?"